mods/favorites-in-navpane.wh.cpp

```diff
@@ -104,7 +104,8 @@ HRESULT WriteOrderRegistry(IStream *pStream)
                 if (SUCCEEDED(hr))
```

Comment:
This should be:
```suggestion
                if (hr == ERROR_SUCCESS)
```
And `hr` is a misleading name here as it's not an `HRESULT`, it's a classic `GetLastError`-like error.